README.md

```diff
@@ -22,7 +22,7 @@ Please complete the instructions below:
 
 1. Create an organization-owned copy of this repository using the [Use this template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) button. 
 2. Create a personal access token (PAT) of type **classic tokens** with the `read:org` and `write:org` scope selected.
-3. Store the PAT as an [organization secret](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces#adding-secrets-for-an-organization) titled `EXERCISE_PAT`. We recommend making the secret available only to your newly created repository.
+3. Store the PAT as an [organization secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) titled `EXERCISE_PAT` in actions tab. We recommend making the secret available only to your newly created repository.
```

Comment:
Storing the PAT as organization secret should be for a repository on actions tab and not for codespaces #26